cmd/config/config.go

```diff
@@ -0,0 +1,54 @@
+package config
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fatih/color"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/candrewlee14/webman/cmd/config/add"
+	"github.com/candrewlee14/webman/cmd/config/remove"
+	"github.com/candrewlee14/webman/config"
+	"github.com/spf13/cobra"
+)
+
+var ConfigCmd = &cobra.Command{
+	Use:   "config",
+	Short: "subcommands for webman config",
+	Long: `
+
+The "config" subcommand allows you to change your base webman config.
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cfg, err := config.Load()
+		if err != nil {
+			return err
+		}
+
+		q := &survey.Input{
+			Message: "Refresh interval",
+			Help:    "How long before webman should refresh a package repository",
```

Comment:
Maybe give an example string here?
Looks you do validate the input after which is good, but an example would probably help.

Reply:
Done in https://github.com/candrewlee14/webman/pull/38/commits/709dd5c5e5398804ca691435f8cc53f47c56693a